### DISCUSSION TITLE
Does v2.3.3 support Chrome debugger?

### DISCUSSION THREAD

**Discussion by theangeloumali** [software-mansion/react-native-reanimated#3047] [Q&A]
Does 2.3.3 support Chrome debugger?

**Comment by SMhdAsadi** [comment]
As documentation says: 

> Please note that Reanimated 2 doesn't support remote debugging, only Flipper can be used for debugging.

So I think you can only use Flipper for debugging.

**Comment by AndreiCalazans** [comment]
Despite the documentation clearly stating that, I do see some commits where this seemed to have been fixed? Sorry to directly tag you here @piaskowyk, but what's the real story behind Chrome debugger support? On 2.9.1 I'm unable to connect to remote debugger am I missing something or is it officially unsupported?
